src/katsdpcontroller/aggregate_sensors.py

```diff
@@ -144,6 +144,22 @@ def aggregate_compute(self) -> Tuple[Sensor.Status, bool]:
         status = Sensor.Status.NOMINAL if synchronised else Sensor.Status.ERROR
         return (status, synchronised)
 
+    def update_aggregate(
+        self,
+        updated_sensor: Optional[Sensor[_T]],
+        reading: Optional[Reading[_T]],
+        old_reading: Optional[Reading[_T]],
+    ) -> Optional[Reading[bool]]:
+        updated_reading = super().update_aggregate(updated_sensor, reading, old_reading)
+        if updated_reading is not None and updated_reading.value == self.value:
+            # Value hasn't changed, just check if the status has
+            if updated_reading.status == self.status:
+                return None
+            else:
+                return updated_reading
+        else:
+            return updated_reading
```

Comment:
You know, I was thinking about stringing together a long if-statement. I chickened out because I thought it wasn't quite pythonic/mypy would complain.